content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -6,141 +6,166 @@ metaDescription: 'Learn how to deploy a TypeScript application to Deno Deploy.'
 
 <TopBlock>
 
-In this guide, you'll learn how to build and deploy a simple application to Deno Deploy. The application uses Prisma to save every request to a PostgreSQL database for inspection later.
+With this guide, you can learn how to build a simple application and deploy it to [Deno Deploy](https://deno.com/deploy). The application uses Prisma to save a log of each request to a PostgreSQL database.
 
-This guide covers Prisma CLI with Deno CLI, Prisma Client, Data Proxy, and Deno Deploy.
+This guide covers the use of Prisma CLI with Deno CLI, Prisma Client, Data Proxy, and Deno Deploy.
 
 <Admonition type="info">
 
-This guide demonstrates how to deploy an application to Deno Deploy in conjunction with a PostgreSQL database, but you can use [any database type supported by Prisma](/reference/database-reference/supported-databases).
+This guide demonstrates how to deploy an application to Deno Deploy in conjunction with a PostgreSQL database, but you can use [any database type that Prisma supports](/reference/database-reference/supported-databases).
 
 </Admonition>
 
 </TopBlock>
 
 ## Prerequisites
 
-- Free [Deno Deploy](https://deno.com/deploy) account
-- Free [GitHub](https://github.com/) account
-- A PostgreSQL database
-- Node.js & NPM installed
-- Git installed
-- [Deno installed](https://deno.land/manual@v1.26.2/getting_started/installation)
+- a free [Deno Deploy](https://deno.com/deploy) account
+- a free [GitHub](https://github.com/) account
+- a PostgreSQL database
+- Node.js & npm installed
+- git installed
+- [Deno installed](https://deno.land/manual@v1.29.1/getting_started/installation)
```

Comment:
Do they have a version-independent link? That would be optimal.

Reply:
I don't think they do. I've just opened this: https://github.com/denoland/manual/issues/516